lib/vcloud/launcher/launch.rb

```diff
@@ -5,11 +5,11 @@ class Launch
       class MissingPreambleError < RuntimeError ; end
       class MissingConfigurationError < RuntimeError ; end
 
-      attr_reader :config, :cli_options
+      attr_reader :config, :options
 
-      def initialize(config_file, cli_options = {})
```

Comment:
:+1: to @dcarley's comment but I also agree with @mattbostock that it should be a separate PR either way